examples/settings.py

```diff
@@ -95,6 +98,23 @@ def make_cfg(settings):
         ortho_sensor_spec.sensor_subtype = habitat_sim.SensorSubType.ORTHOGRAPHIC
         sensor_specs.append(ortho_sensor_spec)
 
+    if settings["fisheye_sensor"]:
```

Comment:
`fisheye_color_sensor`? or `fisheye_rgb_sensor`? 
Later we can have `fisheye_depth_sensor` `fisheye_semantic_sensor`.

Reply:
Yeah, but I kinda want to redo this entire system as it is right now because we also should do the same with the ortho view.

Reply:
Yeah, I would really appreciate it if you can refactor the code here. You can still make this one right in this PR though.
That means when you refactor, you would have an example to follow, and one less place to fix.

Reply:
Came up with a nice clean way of doing it.